www/js/goals.js

```diff
@@ -864,6 +871,81 @@ angular.module('emission.main.goals',['emission.services', 'emission.plugin.logg
         });
     }
 
+    var showUserId = function() {
+        console.log("Showing user id");
+        $ionicPopup.show({
+          title: 'Bic2Cal Survey',
+          templateUrl: 'templates/goals/uid.html',
+          scope: $scope,
+            buttons: [{
+              text: 'Copy user id and open survey',
+              type: 'button-positive',
+              onTap: function(e) {
+                $cordovaClipboard.copy($scope.profile.id).then(function () {
```

Comment:
here, `$cordovaClipboard.copy($scope.profile.id)` should be copying the id to the clipboard when the user clicks, but it doesn't. You can manually copy and paste it, but I can't figure out why this isn't working.